.github/workflows/macos-arm.yml

```diff
@@ -0,0 +1,97 @@
+name: ITK.macOS.Arm64
+
+on:
+    push:
+        branches:
+            - master
+            - 'release*'
+        paths-ignore:
+            - '*.md'
+            - LICENSE
+            - NOTICE
+            - Documentation/*
```

Comment:
You may want a `**` for some of these to match recursively?

Reply:
:+1: done